api/src/main/java/org/openmrs/module/sync/api/impl/SyncIngestServiceImpl.java

```diff
@@ -551,7 +551,12 @@ else if (!alreadyExists && isDelete) {
                         log.debug("xmlFieldValue = " + xmlFieldValue);
                         log.debug("xmlFieldType = " + xmlFieldType);
                     }
-	                SyncUtil.setProperty(o, field, xmlFieldName, xmlFieldValue, xmlFieldType);

```

Comment:
Then when ingesting a sync record, if the value of property is found to be "[null]", then set the property value to null in the target object.